website/docs/language/upgrade-guides/index.mdx

```diff
@@ -5,9 +5,120 @@ description: Upgrading to Terraform v1.3
 
 # Upgrading to Terraform v1.3
 
--> Do you need the upgrade guide for an earlier version of Terraform? Use the
-version selector in the navigation bar to select the version you are intending
-to upgrade to.
+-> Do you need the upgrade guide for a different version of Terraform? Use the version selector in the navigation bar to select the version you are intending to upgrade to.
```

Comment:
We've been trying to steer away from saying "navigation bar" (instructions for the devdot migration) but I think we do need to call out the version selector here :) I'm also just wondering if we can make this note shorter -- you think this is okay or do you think we need to be more explicit?

Another thing that Judith and I thought of today that I wanted to flag for you --> Will anyone be jumping up more than one version? If so, can they come straight to this one or are there versions in between that they need to go through the migration process first. Basically - anything folks should know who are still on like v1.1 or earlier? 

```suggestion
-> **Note:** Use the version selector to view the upgrade guides for older Terraform versions. 
```

Reply:
It's okay! We can fix forward! I'm not that concerned about this note, although maybe we should go back like ONE more to v1.2? 

I'm okay with not including anything, I more poked at it to just make sure we thought of it/see what the response was. I guess my ask is --> if it ever becomes a time when we're like "okay so you really DO need to upgrade to version X before this one if you haven't already", can we just say that to users? I'll trust your judgment about when/if that is ever needed. But I like less text, so if it's not helpful/relevant to include right now, let's not :)

Reply:
I think a specific goal of the 1.x series is that it should always be possible and easy for most users to just jump to the latest 1.x without worrying about intermediate steps. Of course these upgrade guides do capture some situations where a small number of users may need to take a specific action when upgrading, but we're aiming to keep these small enough in scope that relatively few people will actually _need_ the upgrade guides.

In the worst case a subset of the already-smaller number of folks who needed the upgrade guides will not find the upgrade guides and will ask a question in a community forum space. Due to this being a subset of an already-small subset of our users, I think we've done okay with just helping those users find the upgrade guides one-on-one when it arises, and so I think we currently have a good tradeoff for this.

It is possible that one day there is a 2.x release which could once again act as a "stopping point" during upgrades if it has changed significantly enough to require explicit upgrade steps. But at this time we have no plans for any such changes and in fact our current goal is to do exactly the opposite: to continue the 1.x series indefinitely and use backward-compatible or explicit-opt-in techniques to introduce new features and changes. In the event that we change our minds and _do_ find the need or benefit for a 2.x break, I think we can cross that bridge when we get there. :grinning: